python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -920,24 +957,125 @@ async def _check_errors_and_send_events_shutter(self, llc_status: dict[str, typi
             await self.evt_shutterEnabled.set_write(state=EnabledState.FAULT, faultCode=fault_code)
         else:
             await self.evt_shutterEnabled.set_write(state=EnabledState.ENABLED, faultCode="")
-            statuses = llc_status["status"]
-            motion_state: list[str] = []
-            in_position: list[bool] = []
-            # The number of statuses has been validated by the JSON schema. So
-            # here it is safe to loop over all statuses.
-            for status in statuses:
-                translated_status = self._translate_motion_state_if_necessary(status)
-                motion_state.append(translated_status)
-                in_position.append(
-                    translated_status
-                    in [
-                        MotionState.STOPPED,
-                        MotionState.STOPPED_BRAKED,
-                        MotionState.CLOSED,
-                        MotionState.OPEN,
-                    ]
+
+        statuses = llc_status["status"]
+        motion_state: list[str] = []
+        in_position: list[bool] = []
+
+        # The number of statuses has been validated by the JSON schema. So
+        # here it is safe to loop over all statuses.
+        for index, status in enumerate(statuses):
+            translated_status = self._translate_motion_state_if_necessary(status)
+            motion_state.append(translated_status)
+            in_position.append(
+                translated_status
+                in [
+                    MotionState.STOPPED,
+                    MotionState.STOPPED_BRAKED,
+                    MotionState.CLOSED,
+                    MotionState.OPEN,
+                ]
+            )
+            if index == 0:
+                brake_index = Brake.APSCS_LEFT_DOOR
+            else:
+                brake_index = Brake.APSCS_RIGHT_DOOR
+            await self.set_brakes_engaged_bit(translated_status, brake_index)
+        await self.evt_shutterMotion.set_write(state=motion_state, inPosition=in_position)
+        await self.evt_brakesEngaged.set_write(brakes=self.brakes_engaged_bitmask)
+
+    async def _check_errors_and_send_events_rad(self, llc_status: dict[str, typing.Any]) -> None:
+        """Check errors and send events for the rear access door.
+
+        Parameters
+        ----------
+        llc_status : `dict`[`str`, `typing.Any`]
```

Comment:
```suggestion
        llc_status : `dict` [`str`, `typing.Any`]
```
This suggestion would probably be needed in lot of places, so just consider this a heads up rather than a request of change :+1:: https://developer.lsst.io/python/numpydoc.html#dictionary-types.
<img width="706" height="345" alt="image" src="https://github.com/user-attachments/assets/3a9bce27-9fd7-419f-8f9c-5ebba4b278ff" />